stdlib/CMakeLists.txt

```diff
@@ -9,6 +9,11 @@ if(SWIFT_BUILD_RUNTIME_WITH_HOST_COMPILER)
     message(WARNING "Building the swift runtime using the host compiler, and not the just-built clang.")
   endif()
 else()
+  # NOTE(compnerd) save the original compiler for the host swiftReflection that
+  # we build
+  set(_CMAKE_CXX_COMPILER_INITIAL ${CMAKE_CXX_COMPILER})
```

Comment:
Why not just call it CMAKE_CXX_COMPILER_HOST? That would make it clear. Beyond that looks awesome!

Reply:
Naming is not solvable :-p